### ISSUE TITLE
Mute for... dialog inaccessible to screen readers

### ISSUE THREAD

**Issue by TheQuinbox** [UnigramDev/Unigram#2921]
**Describe the bug**
hen clicking mute for... from the menu hile using a screen reader, I get a dialog with 2 unlableed buttons, an OK button, and a Cancel button.
**To Reproduce**
1. Start NVDA
2. Press shift f10 on a chat
3. Select mute -> mute for...
4. Tab
**Expected behavior**
I should be able to use this dialog accessibly
**Version Info**
 - Unigram: 9.2 x64
 - Windows: Windows 11 21H2 (X64) build 22000.1335

**Additional context**
